### ISSUE TITLE
Home app not updating with alarm status

### ISSUE THREAD

**Issue by dswee999** [bryanbartow/homebridge-alarm.com#69]
No matter what I try or how long I wait, the home app never updated the status of the alarm unless it is actually controlled from the Home app. If the alarm is disarmed from the keypad or Alarm.com app, the Home app continues to show that it is still armed. I waited over a day, turned phone off and back, restarted homebridge and Home just continued to show that it was armed. I do have the alarm state API bookmarked in the WrapAPI. What am I missing? Thank you!

**Comment by dswee999** [comment]
I have a z-wave door lock attached to my alarm panel. It's status updates fairly immediately I've noticed, however the actual alarm status never updates. Does anyone know if this is something that can be corrected? Thanks

**Comment by amitgandhinz** [comment]
I was going to create a new issue for this, but i think im seeing the same as you:

Issue: when (re)starting homebridge (eg after updating to the new wrap api login bookmark) and to version 1.0.0, the initial state for the alarm panel is showing up as "Home", instead of "Off" in the Home App.  If i change the state on the physical panel or on the alarm.com webpage (eg Arm Stay), the home app will not detect the new state either (shows "Updating" but then reflects the incorrect state).

Expected Result: reflect the actual current state of the alarm panel by reading from alarm.com (in my case - OFF).

**Comment by amitgandhinz** [comment]
looks like this issue is being discussed towards the bottom of #67

**Comment by dswee999** [comment]
@amitgandhinz yes, I think the first issued turned into a discussion about this issue. I'm not a programmer but I've tried various different things with the WrapAPI and can't get anything to work. Have you had any luck? @bryanbartow @jdshkolnik are either of you experiencing the same issue?

**Comment by jdshkolnik** [comment]
I'm currently using the Frontpoint plugin.

**Comment by dswee999** [comment]
@jdshkolnik that would only work if you use Frontpoint alarm system, correct? I'm using LiveWatch.

**Comment by jdshkolnik** [comment]
True, but @bryanbartow no longer uses an alarm.com system and I'm using a different plugin. Any changes to this plugin will require community involvement.

**Comment by dswee999** [comment]
I'm new to Github, any way to "advertise" for someone who knows what they are doing to take a look at it? Thanks

**Comment by omarshahine** [comment]
I'm seeing the same thing. When pane is disarmed my state shows as "home". changing alarm state from the panel and quitting / oppening the Home app doesn't do anything any more.

**Comment by mkormendy** [comment]
You might try checking out my Alarm.com version of the FrontPoint plugin: https://github.com/mkormendy/homebridge-node-alarm-dot-com
